app/components/tools/GetTerminalFilesHandler.tsx

```
@@ -0,0 +1,74 @@
+      return (
+        <ToolBlock
+          key={toolCallId}
+          icon={<FileDown />}
+          action={status === "streaming" ? "Sharing" : "Shared"}
+          target={getFileNames(filesInput?.files || [])}
+          isShimmer={status === "streaming"}
+        />
+      );
+
+    case "output-available":
+      const fileCount = filesOutput?.fileUrls?.length || 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)